server_test.py

```diff
@@ -0,0 +1,109 @@
+""" Service requests for accessible objects """
```

Comment:
Is this file still relevant? I'm guessing it's an old Flask based implementation so we should remove it from the PR.